src/core/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { baseStyles } from './Pagination.baseStyles';
+import { HtmlSpan, HtmlNav, HtmlDiv } from '../../reset';
+import { asPropType } from '../../utils/typescript';
+import { PageInput } from './PageInput/PageInput';
+import { Button } from '../Button/Button';
+
+const externalClassName = 'fi-link--external';
+
+export interface PaginationInputProps {
+  /** Page input action button label for screen readers  */
+  pageInputButtonText: string;
```

Comment:
Just buttonText might work now that this is under the PageInputProps.

Reply:
There is now separate PageInputProps interface